configs/sites/tier1/acorn/packages_intel.yaml

```diff
@@ -31,7 +31,10 @@
       - '@1.2.1 ~mkl'
     py-numpy:
       require::
-      - '@:1.23 ^openblas'
+      - '^[virtuals=lapack,blas] openblas'
```

Comment:
We should probably use this syntax for all sites when we merge this back into develop?

Reply:
Yeah I think this is the best practices syntax